### ISSUE TITLE
[COMP]: Accessibility utilities filename is misspelled

### ISSUE THREAD

**Issue by andrewrubin** [wethegit/component-library#99]
Easy one. Cannot import the file at `styles/accessibility/_accessibility-utilities.scss` because it's misspelled
